docs/meshstack.user-group-provisioning.md

```diff
@@ -143,6 +143,33 @@ To set up the provisioning on AAD side, have a look at [Microsoft's guideline](h
 | Join(" ", [givenName], [surname])                             | name.formatted                             |
 | Attribute which is used as euid in your meshStack             | externalId                                 |
 
+### How to map attributes via expressions
+
+A common patttern we see is that Admin Users do not have email attributes filled. Here is how to create a mapping that sets the email attribute based on an expression.
```

Comment:
Can you add a quick intro for what we mean with "admin users"? Like

"Some organizations requires the use of dedicated user accounts for administration purposes (segregation of duties). Unlike "normal users" used for everyday office work, these "admin users" often  do not have email attributes" etc.

Reply:
Good idea, I added a motivation to the how